### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by Kelvin-M**
Remove useless commands
Use one docker image for all
Remove useless code

### CODE REVIEW

**Review by inalgnu — APPROVED** [review]

**Comment by Kelvin-M** [comment]
@AlexandrePicosson We can disable the build trigger on celerybeat, celeryworker, flower and celery on dockerhub after this PR merged !

**Comment by AlexandrePicosson** [comment]
Already done for `celerybeat` and `celeryworker` this morning I'll do the others after the merge !

**Comment by AlexandrePicosson** [comment]
Out of curiosity, what is the image size after this fusion and optimisation vs before ?

File: charts/substra-backend/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.2.2
+
+- Remove flower as it is not used.
+
```
Comment (AurelienGasser) [inline charts/substra-backend/CHANGELOG.md:6]:
```suggestion
## 1.4.0

- Consolidate docker images into a unique `substrafoundation/substra-backend` image 

## 1.3.1

- Remove unused `flower` deployment/service

```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 
 Backend of the Substra platform
 
-|  substra-backend | celery | 
-| ------------- | ------------- |
-| [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/substrafoundation/substra-backend)](https://hub.docker.com/r/substrafoundation/substra-backend/builds) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/substrafoundation/celery)](https://hub.docker.com/r/substrafoundation/celery/builds) |
+
+|  substra-backend |
+| ------------- |
+| [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/substrafoundation/substra-backend)](https://hub.docker.com/r/substrafoundation/substra-backend/builds) |
```
Comment (AurelienGasser) [inline README.md:8]: We can remove the table \o/

**Comment by Kelvin-M** [comment]
> Out of curiosity, what is the image size after this fusion and optimisation vs before ?

1.16Gb to 364Mb on my computer

File: charts/substra-backend/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: substra-backend
 home: https://substra.org/
-version: 1.3.1
+version: 1.4.1
```
Comment (AurelienGasser) [inline charts/substra-backend/Chart.yaml:4]:
```suggestion
version: 1.4.0
```

**Review by AurelienGasser — CHANGES_REQUESTED** [review]
Awesome cleanup

This needs a substra-tests CI PR

**Comment by Kelvin-M** [comment]
> Awesome cleanup
> 
> This needs a substra-tests CI PR

You're right !

File: README.md (hunk 2 above)
Comment (Kelvin-M) [reply README.md]: Done !

**Review by AurelienGasser — APPROVED** [review]
LGTM 

(pending `substra-tests` PR)

**Comment by Kelvin-M** [comment]
@AurelienGasser  Done here : https://github.com/SubstraFoundation/substra-tests/pull/150

**Review by inalgnu — APPROVED** [review]

**Comment by AurelienGasser** [comment]
e2e tests: :heavy_check_mark:

**Pull request merged**